package.json

```diff
@@ -12,16 +12,19 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
+		"pretest": "yarn build",
 		"test": "jest --collect-coverage",
-		"test:ci": "jest --verbose --ci --no-stack-trace --collect-coverage",
+		"test:ci": "jest --verbose --ci --runInBand --no-stack-trace --collect-coverage",
 		"docs": "typedoc --name discord.js --exclude **/index.ts --exclude **/*.test.ts --includeVersion --json docs/docs.json",
-		"prebuild": "npm run clean",
+		"prebuild": "yarn clean",
 		"build": "lerna run build",
```

Comment:
If you have packages depending on one another that you'll have to take that into account here. For example you have `@discordjs/rest` depending on `@discordjs/core`. `rest` won't build without the type declarations of `core`, which are to be in the `dist` folder of `core`, which doesn't exist until `core` is build, causing `rest` to fail building.

You can do something like:
```
"build": "run-s build:*",
"build:1": "lerna run build --parallel --scope=@discordjs/core",
"build:2": "lerna run build --parallel --ignore=@discordjs/core"
```

`run-s` is from [npm-run-all](https://yarnpkg.com/package/npm-run-all) (which despite it's name does support yarn) and is "run sequentially", it'll first trigger `build:1` then `build:2` in this case. The ignore and scope flags for lerna will ensure packages are build in the proper order.